### PR TITLE
CTOR-915 : Refresh network-acmepacket-snmp

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh network-acmepacket-snmp

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-acmepacket-snmp.md [hunk 1]
```diff
@@ -5,155 +5,183 @@ title: Acme Packet SNMP
...
+|:----------------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Découvre les ressources via un scan réseau SNMP. Installez le connecteur [Generic SNMP](./applications-protocol-snmp.md) pour obtenir la règle de découverte et créez un modificateur pour le modèle d'hôte **Net-Acmepacket-SNMP-custom** |
+
+Rendez-vous sur la [documentation dédiée](/docs/monitoring/discovery/hosts-discovery) pour en savoir plus sur la découverte automatique d'hôtes.
+
+#### Découverte de services
+
+| Nom de la règle                        | Description                                                             |
+|:---------------------------------------|:------------------------------------------------------------------------|
+| Net-Acmepacket-SNMP-Interface-Name     | Découvre les interfaces réseaux et supervise le statut et l'utilisation |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-acmepacket-snmp.md:62]:
```suggestion
| Net-Acmepacket-SNMP-Interface-Name     | Découvre les interfaces réseau et en supervise le statut et l'utilisation |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-acmepacket-snmp.md [hunk 2]
```diff
@@ -5,155 +5,183 @@ title: Acme Packet SNMP
...
+| Net-Acmepacket-SNMP-Policy-Server-Name | Découvre les policy servers externes et supervise leurs statistiques    |
+| Net-Acmepacket-SNMP-Realm-Name         | Découvre les Realms et supervise l'utilisation                          |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-acmepacket-snmp.md:64]:
```suggestion
| Net-Acmepacket-SNMP-Realm-Name         | Découvre les Realms et en supervise l'utilisation                          |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-acmepacket-snmp.md [hunk 3]
```diff
@@ -5,155 +5,183 @@ title: Acme Packet SNMP
...
+| Net-Acmepacket-SNMP-Sip-Name           | Découvre les interfaces Sip et supervise le statut et l'utilisation     |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-acmepacket-snmp.md:65]:
```suggestion
| Net-Acmepacket-SNMP-Sip-Name           | Découvre les interfaces Sip et en supervise le statut et l'utilisation     |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-acmepacket-snmp.md [hunk 4]
```diff
@@ -162,109 +190,623 @@ A regexp filter is available to target a specific interface identifier - ifName
 
 Afin de contrôler votre équipement Acme Packet, le SNMP doit être configuré.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-acmepacket-snmp.md:191]: standard snmp prerequisites + UDP 161

File: pp/integrations/plugin-packs/procedures/network-acmepacket-snmp.md [hunk 5]
```diff
@@ -5,155 +5,182 @@ title: Acme Packet SNMP
...
+> The services listed above are created automatically when the **Net-Acmepacket-SNMP-custom** host template is used.
+
+</TabItem>
+<TabItem value="Not attached to a host template" label="Not attached to a host template">
+
+| Service Alias  | Service Template                          | Service Description                     | Discovery  |
+|:---------------|:------------------------------------------|:----------------------------------------|:----------:|
+| Codec          | Net-Acmepacket-Codec-SNMP-custom          | Check codec transcoding                 |            |
+| Interfaces     | Net-Acmepacket-Interfaces-SNMP-custom     | Check interfaces                        | X          |
+| Policy-Servers | Net-Acmepacket-Policy-Servers-SNMP-custom | Check external policy servers statitics | X          |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-acmepacket-snmp.md:35]:
```suggestion
| Policy-Servers | Net-Acmepacket-Policy-Servers-SNMP-custom | Check external policy server statistics | X          |
```

File: pp/integrations/plugin-packs/procedures/network-acmepacket-snmp.md [hunk 6]
```diff
@@ -162,110 +189,623 @@ A regexp filter is available to target a specific interface identifier - ifName
 
 To control your Acme Packet, the SNMP must be configured.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/network-acmepacket-snmp.md:190]: standard snmp prerequisites

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-acmepacket-snmp.md (hunk 4 above)
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/network-acmepacket-snmp.md:191]:
```suggestion
### Configuration SNMP

L'agent SNMP doit être activé et configuré sur l'équipement. Veuillez vous référer à la documentation officielle du constructeur/éditeur.

### Flux réseau

La communication doit être possible sur le port UDP 161 depuis le collecteur Centreon vers le serveur supervisé.
```

File: pp/integrations/plugin-packs/procedures/network-acmepacket-snmp.md (hunk 6 above)
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/network-acmepacket-snmp.md:190]:
```suggestion
### SNMP Configuration

The SNMP agent must be configured and activated on the host. Please refer to the official documentation from the manufacturer/publisher.

### Network flow

The target resource must be reachable from the Centreon poller on the UDP/161 SNMP port.
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**